packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -581,6 +581,9 @@ class PageView extends StatefulWidget {
   /// child that could possibly be displayed in the page view, instead of just
   /// those children that are actually visible.
   ///
+  /// If the children [List] is going to be mutated,
```

Comment:
Nit: I probably wouldn't link to `[List]` here - maybe just link to `[children]`.